lib/awestruct/extensions/disqus.rb

```diff
@@ -24,7 +24,7 @@ def disqus_comments()
             var disqus_identifier = #{identifier};
             (function() {
               var dsq = document.createElement("script"); dsq.type = "text/javascript"; dsq.async = true;
-              dsq.src = "http://#{site.disqus}.disqus.com/embed.js";
+              dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
```

Comment:
You scared me for a minute. if you expend the diff, you will see that disqus_shortname is a javascript variable.

    var disqus_shortname = '#{site.disqus}';